string.c

```diff
@@ -10450,17 +10450,30 @@ rb_str_unicode_normalized_p(int argc, VALUE *argv, VALUE str)
 /**********************************************************************
  * Document-class: Symbol
  *
- *  <code>Symbol</code> objects represent names and some strings
- *  inside the Ruby
- *  interpreter. They are generated using the <code>:name</code> and
- *  <code>:"string"</code> literals
- *  syntax, and by the various <code>to_sym</code> methods. The same
- *  <code>Symbol</code> object will be created for a given name or string
- *  for the duration of a program's execution, regardless of the context
- *  or meaning of that name. Thus if <code>Fred</code> is a constant in
- *  one context, a method in another, and a class in a third, the
- *  <code>Symbol</code> <code>:Fred</code> will be the same object in
- *  all three contexts.
+ *  <code>Symbol</code> objects are often used like <code>String</code>
+ *  objects to represent names. Symbols are more memory efficient than
+ *  Strings when the same name is used more than once during the
+ *  execution of a programm.
+ *
+ *  Symbols are generated using the using the following literal
+ *  syntaxes:
+ *
+ *  * <code>:name</code>
+ *  * <code>:"name with spaces"</code>
+ *
+ *  You can also convert some Ruby objects to symbols using common
+ *  <code>to_sym</code> conversion methods:
+ *
+ *     "some name"
+ *     => :"some name"
+ *
+ *  Symbols are often used instead of Strings because the same
+ *  <code>Symbol</code> object will be created in memory for a given
```

Comment:
"same will be created" implies multiple Symbols being created for each use. substitute "created" with "used"

Reply:
Fixed. Thanks!